conda/cli/activate.py

```diff
@@ -105,7 +105,7 @@ def get_path(shelldict):
     This returns PATH in the native representation of the shell - not necessarily
     the native representation of the platform
     """
-    return run_in(shelldict["printpath"], shelldict)[0]
+    return os.getenv("CONDA_PATH")
```

Comment:
CONDA_PATH is a temporary variable set by the activate script.  You don't need to set it.  We have to set it because on windows, there is no way to get the value of PATH in python in linux-format (for example, when using MSYS2).

If you are seeing that error, it points to the wrong activate script being installed.  Can you post that file's contents to a gist somewhere?